ocean_provider/utils/provider_fees.py

```diff
@@ -0,0 +1,57 @@
+from web3.main import Web3
```

Comment:
can you please do an isort of these imports?

Reply:
fixed